schema.prisma

```diff
@@ -140,13 +140,14 @@ model Beneficiary {
 }
 
 model CampaignType {
-  id          String         @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  name        String         @db.VarChar(50)
-  slug        String         @unique @db.VarChar(50)
-  description String?        @db.VarChar(200)
-  parentId    String?        @map("parent_id") @db.Uuid
-  parent      CampaignType?  @relation("ParentCategory", fields: [parentId], references: [id])
-  children    CampaignType[] @relation("ParentCategory")
+  id          String               @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  category    CampaignTypeCategory
```

Comment:
@Yakimov1337 I'll add a default value for the column otherwise the db will throw error for the existing records when we run the migration

Reply:
There's a warning in the migration file:

> Added the required column `category` to the `campaign_types` table without a default value. This is not possible if the table is not empty.